config/routes.rb

```diff
@@ -31,8 +31,8 @@
   # switch off subdomain matching when in development
   if Rails.env.test?
   # or to skip www:
-    match "", to: 'wiki#subdomain', constraints: lambda { |r| r.subdomain.present? && r.subdomain != 'www' && r.subdomain != "i" && r.subdomain != "test" && r.subdomain != "new" && r.subdomain != "alpha"}
-    match "*all", to: 'wiki#subdomain', constraints: lambda { |r| r.subdomain.present? && r.subdomain != 'www' && r.subdomain != "i" && r.subdomain != "test" && r.subdomain != "new" && r.subdomain != "alpha"}
+  match "", to: 'wiki#subdomain', constraints: lambda { |r| r.subdomain.present? && r.subdomain != 'www' && r.subdomain != "i" && r.subdomain != "test" && r.subdomain != "new" && r.subdomain != "alpha"}
```

Comment:
Are these changes necessary? This indentation in particular seems to belong the way it was. 

But otherwise this looks great! Can u share a screenshot of the code being shown?